sdk/contentsafety/Azure.AI.ContentSafety/api/Azure.AI.netstandard2.0.cs

```diff
@@ -0,0 +1,17 @@
+namespace Azure.AI
+{
+    public partial class Client
+    {
+        protected Client() { }
+        public Client(string endpoint, Azure.Core.TokenCredential credential, Azure.AI.ClientOptions options = null) { }
+        public virtual Azure.Core.Pipeline.HttpPipeline Pipeline { get { throw null; } }
+    }
```

Comment:
What are the operations? This client seems to do nothing.

Reply:
Merge the discussion about operation missing to one thread and close this one.